### DISCUSSION TITLE
BirdNET-Pi stops processing/analyzing at midnight

### DISCUSSION THREAD

**Discussion by pentode1** [mcguirepr89/BirdNET-Pi#217] [Q&A] [answered]
I installed BirdNET-Pi a few days ago on a 3B+ and it's working great other than that it stops analyzing data at exactly midnight (last analyzed sample is from a few seconds before midnight)

Everything else seems to work, the web interface works, the web terminal works... the streaming audio works so I presume all the sound subsystem and recording is working too.  Even with a reboot, though, it still doesn't process any data.  The only method I've found to resurrect it so far is to wipe all the data, which I'm guessing isn't the way it's intended to be used ;)

In fact if I shut down before midnight and restart in the morning all the data remains intact and functions as expected-  it's only if it's running _at_ midnight, or so it appears.

Anyone have any ideas what's going on?  Cheers, great project!

**Comment by nilspupils** [comment]
Hi, can you provide a log of the problem?

**Reply by pentode1** [reply]
Yes... in fact I just started trying to get scientific about the problem... forgot it was just a linux system :)
 
basically I have 600+ instances of this:

```
10:28:08 systemd[1]: birdnet_analysis.service: Succeeded.
10:28:10 systemd[1]: birdnet_analysis.service: Scheduled restart job, restart counter is at 552.
10:28:10 systemd[1]: Stopped BirdNET Analysis.
10:28:10 systemd[1]: Started BirdNET Analysis.
10:28:14 systemd[1]: birdnet_analysis.service: Succeeded.
10:28:16 systemd[1]: birdnet_analysis.service: Scheduled restart job, restart counter is at 553.
10:28:16 systemd[1]: Stopped BirdNET Analysis.
10:28:16 systemd[1]: Started BirdNET Analysis.
10:28:19 systemd[1]: birdnet_analysis.service: Succeeded.
10:28:21 systemd[1]: birdnet_analysis.service: Scheduled restart job, restart counter is at 554.
10:28:21 systemd[1]: Stopped BirdNET Analysis.
10:28:21 systemd[1]: Started BirdNET Analysis.
10:28:24 systemd[1]: birdnet_analysis.service: Succeeded.
10:28:26 systemd[1]: birdnet_analysis.service: Scheduled restart job, restart counter is at 555.
10:28:26 systemd[1]: Stopped BirdNET Analysis.
```

Sorry about the formatting, don't know what's up with that.

**Reply by mcguirepr89** [reply]
Look at the time-stamp -- your system time is off by 2 hours :)

**Reply by pentode1** [reply]
Hmm... off 2 hours from what?  I'm in Atlantic daylight time now.  That's the correct time for here. ;)

**Reply by mcguirepr89** [reply]
Please look at [the issue I opened for this ](https://github.com/mcguirepr89/BirdNET-Pi/issues/218)-- I thought you were providing output from my request -- disregard the timestamp comment (for now)

**Comment by CaiusX** [comment, marked as answer]
Answered above.
